examples/federated-learning/tff/federated-learning-for-image-classification/container-image/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/jupyter/tensorflow-notebook:tensorflow-2.15.0
+
+WORKDIR "/home/jovyan"
+
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
```

Comment:
Maybe we should define an `ENTRYPOINT` and a default `CMD`?

Reply:
`ENTRYPOINT` and `CMD` are already defined in the base image